fillers/eips/eip3860.py

```diff
@@ -457,6 +457,18 @@ def generate_create_opcode_initcode_test_cases(
     """
     env = Environment()
 
+    call_code = Yul(
+        """
+        {
+            calldatacopy(0, 0, calldatasize())
+            let call_result := call(10000000, \
+                0x0000000000000000000000000000000000000100, \
+                0, 0, calldatasize(), 0, 0)
+            sstore(0, call_result)
```

Comment:
Could we do `sstore(call_result, 1)` to store a value regardless of the output here?

We would then need to check `s[0]==1, s[1]==0` for failing test cases, and `s[0]==0, s[1]==1` for passing ones.

We should also perform the call with a value less than `10000000` (e.g. `5000000`), because `10000000` is the total gas of the transaction, and luckily we are only able to keep executing because the actual gas sent follows this formula:
```
min(call_gas_value, remaining_gas * 63 / 64)
```
And since we are OOG in the call, we are wasting the entire `remaining_gas * 63 / 64`, so we are left with very little gas after `CALL`, and `SSTORE` consumes `20000` so we are safe, but if this changes in the future for any reason, the transaction could fail unnecessarily.